test/views/EditSetViewSpec.scala

```diff
@@ -134,17 +145,93 @@ class EditSetViewSpec extends BaseSpec {
       editSet.entries.size,
       1
     )
+    when(mockGovukTable.apply(Table(anyList))).thenReturn(any)
```

Comment:
It's a bit odd to return `any`; isn't there something more realistic to return. Why doesn't it matter?

Reply:
Confirmed.